sklearnex/tests/test_n_jobs_support.py

```diff
@@ -49,7 +49,7 @@ def _check_n_jobs_entry_in_logs(records, function_name, n_jobs):
         if f"{function_name}: setting {expected_n_jobs} threads" in rec:
             return True
     # False if n_jobs is set and not found in logs
-    return n_jobs is None
+    return n_jobs is None or n_jobs == -1
```

Comment:
What is the reason for this?

Reply:
this is a small bug i discovered with the n_jobs_support test - when you run it in batch with other tests (what happens in CI) it works perfectly, yet, run standalone - it fails on n_jobs=-1. upon little investigation found following:
n_jobs_wrapper in daal4py does not log anything since for -1 case it matches with old thread count which means nothing is to be set so nothing is rly logged, which fails the test since it relies on logs to pass. agreed with @icfaust to add this clause so whenever we have n_jobs=-1 and "return True" on line 50 did not trigger, meaning that no update to n_threads has been done, success is still returned for the check in the test to pass.